dbt-cta/impactive/models/1_cta_full_refresh/_1_cta_full_refresh__models.yml

```diff
@@ -0,0 +1,1240 @@
+      - name: voterbase_id
+        description: ''
+      - name: city
+        description: ''
+      - name: date_of_birth
+        description: ''
+      - name: registration_status
+        description: ''
+      - name: contact_referral_full_name
+        description: ''
+      - name: created_at
+        description: ''
+      - name: team_id
+        description: ''
+      - name: contact_id
+        description: ''
+      - name: zip_code
+        description: ''
+      - name: unit_number
+        description: ''
+      - name: updated_at
+        description: ''
+      - name: sms_opt_in
+        description: ''
+      - name: different_address
+        description: ''
+      - name: voter_registration_status
+        description: ''
+      - name: activity_id
+        description: ''
+      - name: id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: first_name
+        description: ''
+      - name: email
+        description: ''
+      - name: campaign_id
+        description: ''
+      - name: activity_title
+        description: ''
+      - name: referral_link
+        description: ''
+      - name: email_opt_in
+        description: ''
+      - name: address
+        description: ''
+      - name: completed_voter_registration_at
+        description: ''
+      - name: last_name
+        description: ''
+      - name: county_name
+        description: ''
+      - name: contact_referral_id
+        description: ''
+      - name: team_name
+        description: ''
+      - name: exported_at
+        description: ''
+      - name: state_abbrev
+        description: ''
+      - name: phone
+        description: ''
+      - name: _airbyte_ab_id
+        description: ''
+        tests:
+          - not_null
+          - unique
+      - name: _airbyte_emitted_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_normalized_at
+        description: ''
+        tests:
+          - dbt_expectations.expect_column_values_to_be_of_type:
+              column_type: timestamp
+      - name: _airbyte_registrations_export_hashid
+        description: ''
+
+  - name: users_export_base
+    description: ''
+    columns:
+      - name: num_contacts_synced
+        description: ''
+      - name: selected_voterbase_id
+        description: ''
+      - name: city
+        description: ''
+      - name: recruited_by_id
+        description: ''
+      - name: recruited_by
+        description: ''
+      - name: actions_performed
+        description: ''
+      - name: created_at
+        description: ''
+      - name: reports_filled
+        description: ''
+      - name: van_id
+        description: ''
+      - name: updated_at
+        description: ''
+      - name: supplied_zip_code
+        description: ''
+      - name: id
+        description: ''
+        tests:
```

Comment:
hmm i think this is missing tests?

Reply:
`id` is not a primary key, weirdly for this table. it seems to be a `user_id` but the user can be involved in several different campaigns. so for this one, i added a uniqueness test to the hash id.

Reply:
(i added the same uniqueness test to the similar teams export table as well.)

Reply:
ahh okay gotcha! i saw the empty `tests:` and thought it may be missing some but makes sense now thanks!

Reply:
ahh i see what you're saying! i took that empty test field out. one more approval request coming your way.